catalog/format/iceberg-bench/src/jmh/java/org/projectnessie/catalog/formats/iceberg/meta/AvroSerializeBench.java

```diff
@@ -502,7 +502,7 @@ static DataFile toDataFile(
   // GenericManifestFile c'tor removed in Iceberg 1.10, tackled then
   static ManifestFile toManifestFile(IcebergManifestFile file) {
     ToLongFunction<Long> default0L = boxed -> boxed != null ? boxed : 0L;
-    return new GenericManifestFile(
+    return new GenManifestFile(
```

Comment:
Would you mind making these changes (which LGTM) in a "manual" PR rather that a "renovate" PR? It believe it deserves to be track as a normal enhancement commit, but just a version bump.